netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -187,11 +188,11 @@ public int getDefaultPort() {
     this.freezeProtocolNegotiatorFactory = false;
   }
 
-  @CheckReturnValue
   NettyChannelBuilder(
       String target, ProtocolNegotiator.ClientFactory negotiator,
       @Nullable CallCredentials callCreds) {
-    managedChannelImplBuilder = new ManagedChannelImplBuilder(target, callCreds,
+    managedChannelImplBuilder = new ManagedChannelImplBuilder(
+        target, NettyChannelCredentials.create(negotiator), callCreds,
```

Comment:
I thought we'd be passing the _original_ channel credentials, like the specific instance the user provided. I'd be quite worried with splitting the call creds from the channel creds and performing computation on them separately and eventually recombining them. This API in particular is quite confusing, as it isn't clear that the credentials would be fabricated and not include call credentials.

There are some ways I could potentially accept re-constructing the credentials, but it is a harder sell and it seems unnecessary. We shouldn't also say things like "original" channel credentials if they _aren't_ the originals.

Reply:
Done passing the original channel creds.